xchange-core/src/main/java/org/knowm/xchange/dto/marketdata/Trades.java

```diff
@@ -92,7 +103,7 @@ public Trades(
   }
 
   /** @return a Unique ID for the fetched trades */
-  public long getlastID() {
+  public String getlastID() {
```

Comment:
I think the direction of this change is OK. Te last id should have been a string from the start.

However in this form this would be a breaking API change. 

I think we should introduce a new method that will also fix the fact that `getlastID` does not follow the proper naming convention.

```
public String getLastId() {
    return lastId;
}

@Deprecated
public long getlastID() {
  return Long.valueOf(lastId);
}
```